maps/cam_lao_nam/mission.sqm

```diff
@@ -211,104 +212,111 @@ class AddonsMetaData
 			url="https://www.arma3.com";
 		};
 		class Item19
+		{
```

Comment:
Please unload ALL mods before working on the map.
Remove this dependency please.